examples/bri-3/src/bri/zeroKnowledgeProof/agents/ccsmAnchorHash.agent.ts

```diff
@@ -1,7 +1,9 @@
+import * as crypto from 'crypto';
 import { BadRequestException, Injectable } from '@nestjs/common';
 import { INVALID_ANCHOR_HASH_INPUT } from '../api/err.messages';
 import { v4 as uuidv4 } from 'uuid';
 import { CCSMAnchorHash } from '../models/ccsmAnchorHash';
+import { Document } from '../models/document';
 
 @Injectable()
 export class CCSMAnchorHashAgent {
```

Comment:
inputForProofVerificationparam of method throwErrorIfCCSMAnchorHashInputInvalid should be rename to something like documentContent